CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Changelog
+## 1.26.1
+* Enhancement - Set up pause container user the same as proxy container when App Mesh enabled
```

Comment:
Can you add this:
Bug - Fixed a bug where network stats are not presented in container stats https://github.com/aws/amazon-ecs-agent/pull/1932

Reply:
Good catch.